rfc-116-store-attachment-data-in-content-items.md

```diff
@@ -169,24 +169,35 @@ I propose we do this:
     4. Make the field mandatory.
 
 1. Add a *mandatory* `details.featured_attachments` list to
-   publication formats, which is an ordered list of govspeak or HTML
-   strings (using the `#/definitions/multiple_content_types` schema)
-   rendering attachments from `details.attachments`.
+   publication formats, which is an ordered list of strings that each reference
+   an id of an entry in the details.attachments collection.
+
+   The reasons for preferring an id over the previous embedded HTML approach
+   are:
+
+   - less scope for inconsistency in content items - there isn't a risk
+     the HTML doesn't match the attachment data;
+   - removes a risk of unexpected content, since the HTML is a string
+     it can contain any content and may not necessarily reflect a single
+     attachment;
```

Comment:
I think this point can be removed, as implementing this RFC won't remove the possibility of unexpected content. (Someone may still try and put markdown referencing attachments in their Govspeak).

Reply:
As discussed IRL this will remove the chance of there being unexpected HTML in featured_attachments and doesn't refer to body.

Reply:
👍 early morning review before coffee was consumed :)